BUILD.md

```diff
@@ -92,7 +92,7 @@ directories and place them in any location.
     - [2019](https://www.visualstudio.com/vs/downloads/)
   - The Community Edition of each of the above versions is sufficient, as
     well as any more capable edition.
-- [CMake 3.10.2](https://cmake.org/files/v3.10/cmake-3.10.2-win64-x64.zip) is recommended.
+- [CMake 3.16+](https://cmake.org/download/) is recommended.
```

Comment:
For other repositories, we specify an exact version that matches what we use in CI (as it's the only version of CMake we actually test in our builds, and thus the only version of CMake we can guarantee will work).

Reply:
FYI, CI is currently at CMake 3.19.1, but will be moving to 3.23.2.

Reply:
Updated BUILD.md